rewrite-maven/src/test/java/org/openrewrite/maven/MavenParserTest.java

```diff
@@ -2300,7 +2302,514 @@ void malformedPom() {
     }
 
     @Test
+    void plugins() {
+        rewriteRun(
+          pomXml(
+              """
+                    <project>
+                        <groupId>org.openrewrite.maven</groupId>
+                        <artifactId>a</artifactId>
+                        <version>0.1.0-SNAPSHOT</version>
+    
+                        <build>
+                            <plugins>
+                                <plugin>
+                                    <artifactId>maven-compiler-plugin</artifactId>
+                                    <version>3.11.0</version>
+                                    <configuration>
+                                        <release>11</release>
+                                    </configuration>
+                                </plugin>
+                            </plugins>
+                        </build>
+                    </project>
+                """,
```

Comment:
nitpick: I think we have a recipe that minimizes this leading white space; might be good to apply that here to put the closing `"""` at the same column as `</project>`